classes/user/form/RegistrationForm.inc.php

```diff
@@ -91,7 +91,7 @@ function fetch($request, $template = null, $display = false) {
 
 		if ($this->captchaEnabled) {
 			$publicKey = Config::getVar('captcha', 'recaptcha_public_key');
-			$reCaptchaHtml = '<div class="g-recaptcha" data-sitekey="' . $publicKey . '"></div>';
+			$reCaptchaHtml = '<div class="g-recaptcha" data-sitekey="' . $publicKey . '"></div><label for="g-recaptcha-response" style="display:none;">Recaptcha response</label>';
```

Comment:
@israelcefrin I think you can use the [hidden attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden).

Reply:
Hi @jonasraoni, I can use/add hidden to the element on the 3.4 porting. Would you like me to add on OJS 3.3 as well?

Reply:
As you wish. Both ways are going to work, so I'm just sharing knowledge about this attribute 😁

Reply:
I will port it to OJS3.4 and see how it works.